src/components/ha-card.ts

```diff
@@ -9,6 +9,7 @@ import {
 
 class HaCard extends LitElement {
   @property() public header?: string;
+  @property() public heading?: string;
```

Comment:
This is very weird.

Reply:
🤷‍♂ `ha-card` had `header`, `paper-card` uses `heading`... but I guess I could change that while I'm renaming them anyway...

Reply:
Yeah I would prefer that. Once we add an API, we can no longer remove it without pissing off people that make custom cards.

Reply:
add an API? you're doing what now?

Reply:
"api" "parameter"... You say potata, I say tomato.